packages/dirble/PKGBUILD

```diff
@@ -2,21 +2,21 @@
 # See COPYING for license details.
 
 pkgname=dirble
-pkgver=186.c1b7237
+pkgver=1.4.0
 pkgrel=1
 pkgdesc='Fast directory scanning and scraping tool.'
 groups=('blackarch' 'blackarch-webapp' 'blackarch-scanner')
 arch=('x86_64' 'armv6h' 'armv7h' 'aarch64')
 url='https://github.com/nccgroup/dirble'
-license=('MIT')
+license=('GPL3')
 makedepends=('git' 'cargo' 'rust')
 source=('git+https://github.com/nccgroup/dirble.git')
```

Comment:
Change the source pointing to the release version (like `$pkgname::https://github.com/nccgroup/dirble/archive/v$pkgver.tar.gz`)